frontend/app/src/modules/onchain/send/TradeAssetDisplay.vue

```diff
@@ -1,5 +1,6 @@
 <script setup lang="ts">
 import type { TradableAsset } from '@/modules/onchain/types';
+import type BigNumber from 'bignumber.js';
```

Comment:
I think I also introduced a few of these accidentally, due to auto-complete.

I wonder if I can make a custom rule at some point.

Reply:
Probably we can use this? https://eslint.org/docs/latest/rules/no-restricted-imports#paths